node_windows.go

```diff
@@ -0,0 +1,66 @@
+package restic
+
+import (
+	"os"
+	"syscall"
+)
+
+func (node *Node) OpenForReading() (*os.File, error) {
+	file, err := os.OpenFile(node.path, os.O_RDONLY, 0)
```

Comment:
This can be shortened, the second call to `os.OpenFile()` will most certainly fail exactly as the first.